### PR TITLE
Check URI assignment on POST

### PR DESCRIPTION
**Pull request by kjetilk**
https://solidproject.org/TR/protocol#server-post-uri-assignment et al

### CODE REVIEW

File: protocol/read-write-resource/post-uri-assignment-slug.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Feature: Assignment of URIs for POST requests with Slug
+
+  Background: Set up container
+    * def testContainer = rootTestContainer.createContainer()
+    
+  Scenario: Allow resource creation via POST with Slug
+    * def containerUrl = testContainer.url
+    Given url containerUrl
+    And headers clients.alice.getAuthHeaders('POST', containerUrl)
```
Comment (edwardsph) [inline protocol/read-write-resource/post-uri-assignment-slug.feature:9]:
```suggestion
    Given url testContainer.url
    And headers clients.alice.getAuthHeaders('POST', testContainer.url)
```
Now that we are using the bean properties I wonder if it is cleaner to drop the variable definition and just use the property in each location. What do you think?

File: protocol/read-write-resource/post-uri-assignment.feature [hunk 2]
```diff
@@ -0,0 +1,15 @@
+Feature: Assignment of URIs for POST requests
+
+  Background: Set up container
+    * def testContainer = rootTestContainer.createContainer()
+    
+  Scenario: Allow resource creation via POST
+    * def containerUrl = testContainer.url
+    Given url containerUrl
+    And headers clients.alice.getAuthHeaders('POST', containerUrl)
```
Comment (edwardsph) [inline protocol/read-write-resource/post-uri-assignment.feature:9]:
```suggestion
    Given url testContainer.url
    And headers clients.alice.getAuthHeaders('POST', testContainer.url)
```
as above - if you agree

**Review by edwardsph — APPROVED** [review]
LGTM - with trivial suggestion

**Comment by kjetilk** [comment]
Great, thanks, @edwardsph ! Indeed, I was thinking that might be a good idea, but I wasn't sure about what best practice would be, so I didn't. :-)

**Pull request merged**